common.sh

```diff
@@ -203,15 +203,36 @@ check_provider_os()
     fi
 }
 
+determine_ofi_source_ver()
```

Comment:
Can this be avoided if we know the that the branch is v1.8.x for PR testing?

Reply:
Ah, I did not know that we pass TARGET_BRANCH as an environmental variable to the script, that would make things much easier. I have updated the PR.